templates/title.mustache

```diff
@@ -20,16 +20,16 @@
 				</a>
 			</li>
 			<li>
-			<a class="github-button nav-social" href="https://github.com/canjs/{{name}}"
-				data-count-href="/canjs/{{name}}/stargazers"
-				data-count-api="/repos/canjs/{{name}}#stargazers_count">Star</a>
-	  	</li>
+				<a class="github-button nav-social" href="https://github.com/canjs/{{repoName}}"
+				   data-count-href="/canjs/{{repoName}}/stargazers"
+				   data-count-api="/repos/canjs/{{repoName}}#stargazers_count">Star</a>
+			</li>
 		</ul>
 	{{/if}}
 	{{#with (getClosestWithPackage)}}
 		<ul class="title-links">
 			<!-- <li><a href="#">docco</a></li> -->
-			{{#with (sourceLink .)}}<li><a href="{{.}}">Edit on GitHub</a></li>{{/with}}
+			<li><a href="{{sourceLink (repoName)}}">Edit on GitHub</a></li>
```

Comment:
FYI, those group pages do exist: https://canjs.com/doc/can-connect.behaviors.html

I’m not sure why the edit button doesn’t render now (I think `#with` is always supposed to render), but it wouldn’t hurt to check if the changes in this PR affect that.